util/topgen/intermodule.py

```diff
@@ -6,8 +6,31 @@
 import logging as log
 from typing import Dict, Tuple
 from collections import OrderedDict
+from enum import Enum
 
 from reggen.validate import check_int
+from topgen import lib
+
+IM_TYPES = ['uni', 'req_rsp']
+IM_ACTS = ['req', 'rsp', 'rcv']
+IM_CONN_TYPE = ['1-to-1', '1-to-N', 'broadcast']
+
+
+class ImType(Enum):
+    Uni = 1
+    Bi = 2
```

Comment:
since the enums are `uni` and `bidi`, does it make sense to have the hjson `type` field match this also? We have `req_rsp` for the `bidi` type right now.

I do think `req_rsp` is clearer, but just wondering if it's even better to be consistent.

Reply:
I don't use this Enum class yet. Maybe.. `req_rsp` is clearer.. I will change Enum field too